ddl/failtest/fail_db_test.go

```diff
@@ -119,14 +119,21 @@ func (s *testFailDBSuite) TestHalfwayCancelOperations(c *C) {
 	// Test schema is correct.
 	tk.MustExec("select * from t")
 	// test for renaming table
-	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/renameTableErr", `return(true)`), IsNil)
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/renameTableErr", `return("ty")`), IsNil)
 	defer func() {
 		c.Assert(failpoint.Disable("github.com/pingcap/tidb/ddl/renameTableErr"), IsNil)
 	}()
 	tk.MustExec("create table tx(a int)")
 	tk.MustExec("insert into tx values(1)")
 	_, err = tk.Exec("rename table tx to ty")
 	c.Assert(err, NotNil)
+	tk.MustExec("create table ty(a int)")
+	tk.MustExec("insert into ty values(2)")
+	_, err = tk.Exec("rename table ty to tz, tx to ty")
+	c.Assert(err, NotNil)
+	_, err = tk.Exec("rename table tx to ty, ty to tz")
+	c.Assert(err, NotNil)
```

Comment:
After executing `rename table ty to tz, tx to ty`, does `ty` exist? Does `tz` exist?

Reply:
It's nice to have this check. I have added a check for it.

Reply:
Do we need to check kind of error returned in `ty to tz`?